libs/wingsdk/src/fs/fs.ts

```diff
@@ -221,8 +221,15 @@ export class Util {
    */
   public static readYaml(filepath: string): Json[] {
     const text = Util.readFile(filepath);
-    const yamlObjs = yaml.parseAllDocuments(text);
-    return yamlObjs.map((o: any) => JSON.parse(JSON.stringify(o)) as Json);
+    const yamlDocs = yaml.parseAllDocuments(text);
+    return yamlDocs.map((doc) => {
+      if (doc.contents && doc.contents.toJSON) {
+        return doc.contents.toJSON();
+      } else {
+        console.warn("Unexpected document structure:", doc);
```

Comment:
Why not throw?

Reply:
I assumed you'd prefer to continue processing the remaining documents in the file instead of stopping the process altogether.